client/src/components/TopSongGallaryStyle.jsx

```diff
@@ -0,0 +1,136 @@
+import styled from "styled-components";
+
+export const TopHSongCountainer = styled.div`
+border-radius: 8px;
+@media(min-width: 350px) and (max-width: 700px) {
+  width: 98vw;
```

Comment:
don't use vh/vw, they make problems in web mobile

Reply:
done